### DISCUSSION TITLE
Example of megamenu navbar

### DISCUSSION THREAD

**Discussion by ritchiemarkov** [bootscore/bootscore#407] [Q&A] [answered]
Hello Basti,

Can you please provide an example of megamenu navbar because it's a bit unclear for me to follow steps described on the theme site.
 I have followed steps by megamenu-navwalker but had some troubles on my back end.

Ty

**Comment by ritchiemarkov** [comment]
Sorry my bad. Just seen a topic on 3 level menu. Gonna try it now.

**Reply by crftwrk** [reply]
Yes, this one https://github.com/bootscore/bootscore/discussions/347

**Comment by ritchiemarkov** [comment, marked as answer]
Ok, Desktop is fine. But i dont understand why my off canvas menu is keep closing. no dropdown shown. Is there any workaround or i missed something.

**Reply by crftwrk** [reply]
Closing script has changed in last release. Update to latest theme version and offcanvas stays open when click on nested dropdown-toggle https://github.com/bootscore/bootscore/releases/tag/v5.2.3.2.

Check here https://github.com/bootscore/bootscore/pull/350

**Reply by ritchiemarkov** [reply]
Ty so much. <3
